node/src/components/consensus/era_supervisor.rs

```diff
@@ -996,15 +997,24 @@ where
                 self.disconnect(sender)
             }
             ProtocolOutcome::CreatedGossipMessage(out_msg) => {
+                let message = ConsensusMessage::Protocol {
+                    era_id,
+                    payload: out_msg,
+                };
                 // TODO: we'll want to gossip instead of broadcast here
                 self.effect_builder
-                    .broadcast_message(era_id.message(out_msg).into())
+                    .broadcast_message(message.into())
+                    .ignore()
+            }
+            ProtocolOutcome::CreatedTargetedMessage(out_msg, to) => {
```

Comment:
We might as well just call that `payload` (also in `CreatedGossipMessage`):
```suggestion
            ProtocolOutcome::CreatedTargetedMessage(payload, to) => {
```